tdm-be/src/app.ts

```diff
@@ -17,7 +18,13 @@ import swaggerFile from '~/swagger/swagger-config.json';
 
 const app = express();
 
-const port = environment.port;
+const limiter = rateLimit({
+    windowMs: 60 * 1000, // 1 minute
+    limit: 1000, // Limit each IP to 1000 requests per `window` (here, per minute)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** 16 per second?

Isn't it still too much?
I mean: the application is built to be used by humans, not machines.‏

Reply:
Yes, but I don't know what to put in it. We need to discuss it more.

16 per second is still a lot, 960 per minute is almost the same as what I set before.

We need to take into account the file distribution (assets and vendors) and the API call, I think the best way is to have a larger window time with a respectable number of resquests.